### PR TITLE
Update cli.py

### PR DESCRIPTION
**Pull request by henrymbuguakiarie**
## Purpose

I've added a named snippet app instance section within the code for out-of-repo snippet references.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by henrymbuguakiarie** [comment]
Hello @ckittel,

I've added named snippet where we are creating the confidential app instance. Please review and merge.

Cc: @mmacy, @Dickson-Mwendia

File: src/cli-access-protected-api/cli.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 # MSAL configs
 # <ms_docref_configure_msal>
 config = {
-    # Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+    # Full directory URL, in the form of https://login.microsoftonline.com/<tenant_id>
```
Comment (ckittel) [inline src/cli-access-protected-api/cli.py:22]: If you change it here, you need to also change it in the README.md file. They are supposed to stay in sync. So either revert this change or sync the README, either is fine.

**Review by ckittel — CHANGES_REQUESTED** [review]
One small item, then clear to merge

File: src/cli-access-protected-api/cli.py (hunk 1 above)
Comment (henrymbuguakiarie) [reply src/cli-access-protected-api/cli.py:22]: Hello @ckittel I've updated the README.md file as well. Thanks for this guide.

Comment (ckittel) [reply src/cli-access-protected-api/cli.py:22]: Thanks, and good catch on the `app.py` -> `cli.py`!

**Review by ckittel — APPROVED** [review]
🚀
